hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
```

Comment:
I would set the `cutoff` to `None` by default as any uninformed guess will always be wrong and it can be misleading to the unaware user who may think that we have a clever way to do the job for him.

Talking about clever ways of estimating the number of PCs, maybe @tjof2 knows of any worth using? (I've tested the one in sklearn and it is so slow that I killed the computation before finding out if it was doing its job properly...)